crates/virtio-queue/src/mock.rs

```diff
@@ -371,16 +397,60 @@ impl<'a, M: GuestMemory> MockSplitQueue<'a, M> {
         q
     }
 
-    /// Writes a single descriptor chain to the memory object of the queue, at the beginning of the
+    /// Writes descriptors to the memory object of the queue, at the beginning of the
     /// descriptor table, and returns the associated `DescriptorChain` object.
-    pub fn build_desc_chain(&self, descs: &[Descriptor]) -> DescriptorChain<&M> {
-        self.add_desc_chain(descs, 0);
+    pub fn build_chain_of_descriptors(
```

Comment:
This function looks like a generalization of `build_desc_chain`. We should think about a way to just keep one of them.

Reply:
Done. I only kept build_desc_chain and added a bool parameter that, when true, only one descriptor chain is built and, when false, multiple descriptor chains are built.